app/config/router.js

```diff
@@ -122,7 +124,7 @@ export const Tabs = createBottomTabNavigator({
       tabBarLabel: 'Daily News',
       tabBarIcon: ({ tintColor }) => <Image
         source={require('../media/Daily.png')}
```

Comment:
Didn't you delete Daily.png in this commit?

Reply:
> Didn't you delete Daily.png in this commit?

No. https://github.com/TheStanfordDaily/StanfordDaily_Mobile/pull/131/files#diff-a8b25c28e47737c12c83a4609d35b7df